core/domain/rights_manager.py

```diff
@@ -160,6 +177,19 @@ def _save_activity_rights(
         commit_cmds):
     """Saves an ExplorationRights or CollectionRights domain object to the
     datastore.
+
+    Args:
+        committer_id: str. ID of the committer.
+        activity_rights: ActivityRights. Rights object for the activity.
+        activity_type: The type of activity. Either one of these:
+            feconf.ACTIVITY_TYPE_EXPLORATION
+            feconf.ACTIVITY_TYPE_COLLECTION
+        commit_message: str. Descriptive message for the commit
+        commit_cmds: list. A list of commands describind kind of commit was
```

Comment:
Usually we write list(str) or list(dict) or list(...). Please could you augment the type information?

Also, "describind kind of commit" --> "describing what kind of commit"